e2e/multi-app-project/back-end/Dockerfile

```diff
@@ -1,6 +1,24 @@
-FROM nginx
+# We specify the base image we need for our
+# go application
+FROM golang:1.13
+# We create an /app directory within our
+# image that will hold our application source
+# files
+RUN mkdir /app
+# We copy everything in the root directory
+# into our /app directory
+ADD . /app
+# We specify that we now wish to execute
+# any further commands inside our /app
+# directory
+WORKDIR /app
+# Avoid the GoProxy
+ENV GOPROXY=direct
```

Comment:
Does this apply to all users or just to us who use the internal connect?

Reply:
This is just an annoying thing we have to do for our E2E tests to work when we're on the amazon network.